features/learn/src/main/kotlin/com/teacherworkout/features/learn/data/impl/FakeLessonsRepository.kt

```diff
@@ -0,0 +1,73 @@
+package com.teacherworkout.features.learn.data.impl
+
+import com.teacherworkout.features.learn.R
+import com.teacherworkout.features.learn.model.LessonTheme
+import com.teacherworkout.features.learn.data.LessonsRepository
+import com.teacherworkout.features.learn.data.Result
+import kotlinx.coroutines.delay
+
+class FakeLessonsRepository: LessonsRepository {
+    val dummyLessonThemes = listOf(
+            LessonTheme(
+                R.drawable.art1,
+                "Dictie si vorbire"
+            ),
+            LessonTheme(
+                R.drawable.art2,
+                "Interactiunea la clasa"
+            ),
+            LessonTheme(
+                R.drawable.art3,
+                "Gestionarea crizelor"
+            ),
+            LessonTheme(
+                R.drawable.art4,
+                "ADHD"
+            ),
+            LessonTheme(
+                R.drawable.art1,
+                "Incluziune"
+            ),
+            LessonTheme(
+                R.drawable.art3,
+                "Dizabilitati motorii"
+            ),
+            LessonTheme(
+                R.drawable.art2,
+                "Cum discuti despre boli psihice"
+            ),
+            LessonTheme(
+                R.drawable.art1,
+                "Dictie si vorbire"
+            ),
+            LessonTheme(
+                R.drawable.art2,
+                "Interactiunea la clasa"
+            ),
+            LessonTheme(
+                R.drawable.art3,
+                "Gestionarea crizelor"
+            ),
+            LessonTheme(
+                R.drawable.art4,
+                "ADHD"
+            ),
+            LessonTheme(
+                R.drawable.art1,
+                "Incluziune"
+            ),
+            LessonTheme(
+                R.drawable.art3,
+                "Dizabilitati motorii"
+            ),
+            LessonTheme(
+                R.drawable.art2,
+                "Cum discuti despre boli psihice"
+            ),
+        )
+
+    override suspend fun getAllLessonThemes(): Result<List<LessonTheme>> {
+        delay(1000)
```

Comment:
Why are we adding this delay here?

Reply:
It's a fake repository. In the real implementation we'll have some delay in there.